src/security_utilities_rust/src/microsoft_security_utilities_core/identifiable_secrets.rs

```diff
@@ -1,21 +1,36 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+use base_62;
 use base64::{engine::general_purpose, Engine as _};
+use chrono::Datelike;
 use core::panic;
+use lazy_static::lazy_static;
 use std::{mem};
 use super::*;
 use rand::prelude::*;
 use rand_chacha::ChaCha20Rng;
 use regex::Regex;
 use substring::Substring;
 
+lazy_static! {
+    pub static ref VERSION_TWO_CHECKSUM_SEED: u64 = compute_his_v1_checksum_seed("Default0");
+}
+
+pub static COMMON_ANNOTATED_KEY_REGEX_PATTERN: &str = r"(?-i)[A-Za-z0-9]{52}JQQJ99[A-Za-z0-9][A-L][A-Za-z0-9]{16}[A-Za-z][A-Za-z0-9]{7}([A-Za-z0-9]{2}==)?";
```

Comment:
>JQQJ99 [](http://example.com/codeflow?start=76&length=6)

See my recent PR, this should be. The `D` in this regex indicates a key `D`erived from a parent common annotated key.

`JQQJ9(9|D)`